PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/utility/AccessPrivate.java

```diff
@@ -1,4 +1,4 @@
-package nl.tudelft.pl2016gr2.test.utility;
```

Comment:
About the checkstyle in the annotations classes, they causes the build to fail because of a line length bigger than 100 characters, not because I changed the package name. Furthermore, I build it locally but the build succeeded.